BrainPortal/app/models/portal_task.rb

```diff
@@ -605,20 +616,49 @@ def empty? #:nodoc:
       @real_errors.empty?
     end
 
+    def full_message(*args) #:nodoc:
+      @real_errors.full_message(*args)
+    end
+
     def full_messages(*args) #:nodoc:
       @real_errors.full_messages(*args)
     end
 
-    def generate_message(paramspath,*args) #:nodoc:
-      @real_errors.generate_message(paramspath.to_la_id,*args)
+    def get(paramspath) #:nodoc:
+      @real_errors.get(paramspath.to_la_id.parameterize.underscore.to_sym)
+    end
+
+    def has_key?(paramspath) #:nodoc:
+      @real_errors.has_key?(paramspath.to_la_id.parameterize.underscore.to_sym)
+    end
+
+    def include?(paramspath) #:nodoc:
+      @real_errors.include?(paramspath.to_la_id.parameterize.underscore.to_sym)
+    end
+
+    def keys #:nodoc:
+      @real_errors.keys
+    }
```

Comment:
Yes. It returns the keys in the format :cbrain_task_params_xxx. Is that what we want?